src/live-preview.ts

```diff
@@ -426,9 +426,9 @@ export default class LivePreview {
 
     // Request parent for data sync when document loads
     private requestDataSync() {
-        this.handleUserChange({
-            live_preview: "init", // this is the hash of the live previewd
-        });
+        // this.handleUserChange({
+        //     live_preview: "init", // this is the hash of the live previewd
+        // });
```

Comment:
If this code is not required, remove it instead of commenting it out. Will there be any changes in the test cases?

Reply:
removed the code completely, there will be no changes in the test cases